docs/_static/searchtools.js

```diff
@@ -0,0 +1,644 @@
+/**
+ * Return array with titles of ancestors of file.
+ * @param {number} idx - The index of the result item in global list of files
+ * @returns array
+ */
+function _getParentTitles(idx, docNames, titles) {
+  let path = docNames[idx]
+  let parentpathtokens = path.split('/').slice(0, -1);
+
+  let parentTitles = parentpathtokens.map((el, index) => {
+    let foo = `${parentpathtokens.slice(0, index+1).join('/')}`
+    let parentId = docNames.indexOf(foo);
+    if (parentId === -1) {
+      foo = `${parentpathtokens.slice(0, index+1).join('/')}/index`
+      parentId = docNames.indexOf(foo);
+    }
+    let title = parentId === -1 ? title_repository : titles[parentId];
+    return title
+  })
+
+  return parentTitles
+}
+
+/**
+ * Simple result scoring code.
+ */
+if (typeof Scorer === "undefined") {
+  var Scorer = {
+    // Implement the following function to further tweak the score for each result
+    // The function takes a result array [docname, title, anchor, descr, score, filename]
+    // and returns the new score.
+    /*
+    score: result => {
+      const [docname, title, anchor, descr, score, filename] = result
+      return score
+    },
+    */
+
+    // query matches the full name of an object
+    objNameMatch: 11,
+    // or matches in the last dotted part of the object name
+    objPartialMatch: 6,
+    // Additive scores depending on the priority of the object
+    objPrio: {
+      0: 15, // used to be importantResults
+      1: 5, // used to be objectResults
+      2: -5, // used to be unimportantResults
+    },
+    //  Used when the priority is not in the mapping.
+    objPrioDefault: 0,
+
+    // query found in title
+    title: 15,
+    partialTitle: 7,
+    // query found in terms
+    term: 5,
+    partialTerm: 2,
+  };
+}
+
+const _removeChildren = (element) => {
+  while (element && element.lastChild) element.removeChild(element.lastChild);
+};
+
+/**
+ * See https://developer.mozilla.org/en-US/docs/Web/JavaScript/Guide/Regular_Expressions#escaping
+ */
+const _escapeRegExp = (string) =>
+  string.replace(/[.*+\-?^${}()|[\]\\]/g, "\\$&"); // $& means the whole matched string
+
+
+function _getBreadcrumbs(item, linkUrl) {
+  // No breadcrumbs for top level pages
+  if (item[0].split('/')[1] == 'index') {
+    return null
+  }
+  let parentTitles = item[6];
+  
+  let parentDefaultTitles = [
+    "Plone trainings",
+    "Training for Plone developers"
+  ];
+  parentTitles = Array.isArray(parentTitles) ? parentTitles : parentDefaultTitles;      
+  let pathTokens = item[0].split('/')
+    .slice(0, -1);
+  let pathArray = pathTokens.map((el, index) => {
+    return {
+      "path": pathTokens.slice(0, index+1).join('/'),
+      "title": parentTitles[index]
+    }
+  })
+  let markup = pathArray
+    .map((el, idx) => {
+        return `<a href="/${el.path}">${el.title}</a>` 
+      })
+  markup.push(`<span class="lastbreadcrumb">${item[1]}</span>`)
+  return markup.join('<span class="pathseparator">&gt;</span>');
```

Comment:
I think spaces make it easier to read. Compare before and after.

![Screenshot 2025-03-16 at 6 27 06 AM](https://github.com/user-attachments/assets/6025ac12-d0c2-451c-a9fa-4927b2dd6a05)


```suggestion
  return markup.join('<span class="pathseparator"> &gt; </span>');
```